src/main/java/io/strimzi/test/container/StrimziKafkaCluster.java

```diff
@@ -252,18 +345,29 @@ public Collection<GenericContainer<?>> getNodes() {
      */
     @DoNotMutate
     public String getNetworkBootstrapServers() {
-        return nodes.stream()
+        return getBrokers().stream()
                 .map(broker -> ((StrimziKafkaContainer) broker).getNetworkBootstrapServers())
                 .collect(Collectors.joining(","));
     }
 
     @Override
     public String getBootstrapServers() {
-        return nodes.stream()
+        return getBrokers().stream()
             .map(KafkaContainer::getBootstrapServers)
             .collect(Collectors.joining(","));
     }
 
+    /**
+     * Get the bootstrap controllers that can be used for controller operations
+     * @return a comma separated list of Kafka controller endpoints
+     */
+    @Override
+    public String getBootstrapControllers() {
```

Comment:
Should we also have a `getNetworkBootstrapControllers()` method?

Reply:
I have implemented it in `getNetworkBootstrapControllers`, you can check.